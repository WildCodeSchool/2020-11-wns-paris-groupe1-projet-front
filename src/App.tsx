import React from "react"
// import { createStyles, makeStyles, Theme } from "@material-ui/core/styles"
import client from "../src/services/index2"
import UploadLayout from "./components/upload/UploadLayout"
import { ApolloProvider } from "@apollo/client"

// const useStyles = makeStyles((theme: Theme) =>
//   createStyles({
//     root: {
//       flexGrow: 1,
//     },
//     menuButton: {
//       marginRight: theme.spacing(2),
//     },
//     title: {
//       flexGrow: 1,
//     },
//   })
// )

export default function ButtonAppBar() {
  // const classes = useStyles()

  return (
    <ApolloProvider client={client}>
      <div className="App-header" style={{ display: "flex", justifyContent: "center", alignItems: "center" }}>
        <UploadLayout />
      </div>
    </ApolloProvider>
  )
}
