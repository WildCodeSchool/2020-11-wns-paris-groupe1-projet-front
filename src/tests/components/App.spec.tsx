import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from './../../App';

afterEach(cleanup)

describe('App', () => {
  it('should compare App snapshots', () => {
    const { asFragment } = render(<App />)
    const app = asFragment()
    expect(app).not.toBe(null);
    expect(app).toMatchSnapshot();
  });
});