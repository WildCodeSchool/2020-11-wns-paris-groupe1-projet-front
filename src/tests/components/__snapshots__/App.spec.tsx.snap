// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should compare App snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="form"
    >
      <main
        class="MuiContainer-root MuiContainer-maxWidthSm"
      >
        <div
          class="makeStyles-paper-1"
        >
          <h1
            class="MuiTypography-root MuiTypography-h5"
          >
            Connecte toi
          </h1>
          <form
            class="makeStyles-form-2"
            novalidate=""
          >
            <div
              class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined Mui-focused Mui-focused Mui-required Mui-required"
                data-shrink="true"
                for="email"
                id="email-label"
              >
                Adresse email
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth Mui-focused Mui-focused MuiInputBase-formControl"
              >
                <input
                  aria-invalid="false"
                  autocomplete="email"
                  class="MuiInputBase-input MuiOutlinedInput-input"
                  id="email"
                  name="email"
                  required=""
                  type="text"
                  value=""
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-5 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-7 PrivateNotchedOutline-legendNotched-8"
                  >
                    <span>
                      Adresse email *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <div
              class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
                data-shrink="false"
                for="password"
                id="password-label"
              >
                Mot de passe
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              >
                <input
                  aria-invalid="false"
                  autocomplete="current-password"
                  class="MuiInputBase-input MuiOutlinedInput-input"
                  id="password"
                  name="password"
                  required=""
                  type="password"
                  value=""
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-5 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-7"
                  >
                    <span>
                      Mot de passe *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-9 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <input
                    class="PrivateSwitchBase-input-12"
                    data-indeterminate="false"
                    type="checkbox"
                    value="remember"
                  />
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Se souvenir de moi
              </span>
            </label>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-submit-3 MuiButton-fullWidth"
              tabindex="0"
              type="submit"
            >
              <span
                class="MuiButton-label"
              >
                Se connecter
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <div
              class="MuiGrid-root MuiGrid-container"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
              >
                <a
                  class="makeStyles-link-4"
                  href="/sign-in"
                >
                  Mot de passe oublié ?
                </a>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <a
                  class="makeStyles-link-4"
                  href="/sign-up"
                >
                  Si tu n'as pas encore de compte, crée-toi en un !
                </a>
              </div>
            </div>
          </form>
        </div>
        <div
          class="MuiBox-root MuiBox-root-13"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
          >
            Copyright © 
            <a
              href="/sign-in"
            >
              Your Website
            </a>
             2020.
          </p>
        </div>
      </main>
    </div>
  </div>
</DocumentFragment>
`;

exports[`App should compare App snapshots 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="form"
    >
      <main
        class="MuiContainer-root MuiContainer-maxWidthSm"
      >
        <div
          class="makeStyles-paper-1"
        >
          <h1
            class="MuiTypography-root MuiTypography-h5"
          >
            Connecte toi
          </h1>
          <form
            class="makeStyles-form-2"
            novalidate=""
          >
            <div
              class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined Mui-focused Mui-focused Mui-required Mui-required"
                data-shrink="true"
                for="email"
                id="email-label"
              >
                Adresse email
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth Mui-focused Mui-focused MuiInputBase-formControl"
              >
                <input
                  aria-invalid="false"
                  autocomplete="email"
                  class="MuiInputBase-input MuiOutlinedInput-input"
                  id="email"
                  name="email"
                  required=""
                  type="text"
                  value=""
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-5 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-7 PrivateNotchedOutline-legendNotched-8"
                  >
                    <span>
                      Adresse email *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <div
              class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
                data-shrink="false"
                for="password"
                id="password-label"
              >
                Mot de passe
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              >
                <input
                  aria-invalid="false"
                  autocomplete="current-password"
                  class="MuiInputBase-input MuiOutlinedInput-input"
                  id="password"
                  name="password"
                  required=""
                  type="password"
                  value=""
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-5 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-7"
                  >
                    <span>
                      Mot de passe *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-9 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <input
                    class="PrivateSwitchBase-input-12"
                    data-indeterminate="false"
                    type="checkbox"
                    value="remember"
                  />
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Se souvenir de moi
              </span>
            </label>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-submit-3 MuiButton-fullWidth"
              tabindex="0"
              type="submit"
            >
              <span
                class="MuiButton-label"
              >
                Se connecter
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <div
              class="MuiGrid-root MuiGrid-container"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
              >
                <a
                  class="makeStyles-link-4"
                  href="/sign-in"
                >
                  Mot de passe oublié ?
                </a>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <a
                  class="makeStyles-link-4"
                  href="/sign-up"
                >
                  Si tu n'as pas encore de compte, crée-toi en un !
                </a>
              </div>
            </div>
          </form>
        </div>
        <div
          class="MuiBox-root MuiBox-root-13"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
          >
            Copyright © 
            <a
              href="/sign-in"
            >
              Your Website
            </a>
             2020.
          </p>
        </div>
      </main>
    </div>
  </div>
</DocumentFragment>
`;

exports[`App should compare snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="form"
    >
      <main
        class="MuiContainer-root MuiContainer-maxWidthSm"
      >
        <div
          class="makeStyles-paper-1"
        >
          <h1
            class="MuiTypography-root MuiTypography-h5"
          >
            Connecte toi
          </h1>
          <form
            class="makeStyles-form-2"
            novalidate=""
          >
            <div
              class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined Mui-focused Mui-focused Mui-required Mui-required"
                data-shrink="true"
                for="email"
                id="email-label"
              >
                Adresse email
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth Mui-focused Mui-focused MuiInputBase-formControl"
              >
                <input
                  aria-invalid="false"
                  autocomplete="email"
                  class="MuiInputBase-input MuiOutlinedInput-input"
                  id="email"
                  name="email"
                  required=""
                  type="text"
                  value=""
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-5 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-7 PrivateNotchedOutline-legendNotched-8"
                  >
                    <span>
                      Adresse email *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <div
              class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
                data-shrink="false"
                for="password"
                id="password-label"
              >
                Mot de passe
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              >
                <input
                  aria-invalid="false"
                  autocomplete="current-password"
                  class="MuiInputBase-input MuiOutlinedInput-input"
                  id="password"
                  name="password"
                  required=""
                  type="password"
                  value=""
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-5 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-7"
                  >
                    <span>
                      Mot de passe *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-9 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <input
                    class="PrivateSwitchBase-input-12"
                    data-indeterminate="false"
                    type="checkbox"
                    value="remember"
                  />
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Se souvenir de moi
              </span>
            </label>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-submit-3 MuiButton-fullWidth"
              tabindex="0"
              type="submit"
            >
              <span
                class="MuiButton-label"
              >
                Se connecter
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <div
              class="MuiGrid-root MuiGrid-container"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
              >
                <a
                  class="makeStyles-link-4"
                  href="/sign-in"
                >
                  Mot de passe oublié ?
                </a>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <a
                  class="makeStyles-link-4"
                  href="/sign-up"
                >
                  Si tu n'as pas encore de compte, crée-toi en un !
                </a>
              </div>
            </div>
          </form>
        </div>
        <div
          class="MuiBox-root MuiBox-root-13"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
          >
            Copyright © 
            <a
              href="/sign-in"
            >
              Your Website
            </a>
             2020.
          </p>
        </div>
      </main>
    </div>
  </div>
</DocumentFragment>
`;
